alterpartitionreassignments_test.go

```diff
@@ -0,0 +1,58 @@
+package kafka
+
+import (
+	"context"
+	"testing"
+
+	ktesting "github.com/segmentio/kafka-go/testing"
+)
+
+func TestClientAlterPartitionReassignments(t *testing.T) {
+	if !ktesting.KafkaIsAtLeast("2.4.0") {
+		return
+	}
+
+	ctx := context.Background()
+	client, shutdown := newLocalClient()
+	defer shutdown()
+
+	topic := makeTopic()
+	createTopic(t, topic, 2)
+	defer deleteTopic(t, topic)
+
+	// Local kafka only has 1 broker, so any partition reassignments are really no-ops.
+	resp, err := client.AlterPartitionReassignments(
+		ctx,
+		&AlterPartitionReassignmentsRequest{
+			Topic: topic,
+			Assignments: []AlterPartitionReassignmentsRequestAssignment{
+				{
+					PartitionID: 0,
+					BrokerIDs:   []int{1},
+				},
+				{
+					PartitionID: 1,
+					BrokerIDs:   []int{1},
+				},
+			},
+		},
+	)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+	if resp.ErrorCode != 0 {
+		t.Error(
+			"Bad error code in response",
+			"expected", 0,
+			"got", resp.ErrorCode,
+		)
+	}
```

Comment:
Yeah, so I deliberately didn't use testify (as much as I'd like to) because it seems like the vast majority of the tests in the repo don't. If it's ok to use testify, maybe we should migrate all of our existing tests in a separate change?